docs/pages/omnia3/Modeler/omnia3_modeler_components.md

```diff
@@ -0,0 +1,189 @@
+| pageSize     | Inbound   | Integer         | The number of records to be returned                                                  |
+| totalPages   | Outbound  | Integer         | The total number of pages that can be obtained for the current query and page size    |
+| totalRecords | Outbound  | Integer         | The total number of records that can be returned as the result of the query execution |
+| isLoading    | Outbound  | Boolean         | Boolean to indicate if the query is being executed and data is still loading          |
+
+- Events:
+
+| Event       | Description                                                          |
+| ----------- | -------------------------------------------------------------------- |
+| BeforeLoad  | Code executed before making the request to the API                   |
+| OnLoad      | Code executed when data is retrieved from API, before being rendered |
+| OnLoadError | Code executed when the request to retrieve data returns an error     |
+
+- Methods:
+
+| Method    | Description                                          |
+| --------- | ---------------------------------------------------- |
+| refresh   | Method that executes the query to retrieve new data  |
+| exportCSV | Method that requests to the API a data export to CSV |
+
+### entityForm
+
+The entityForm Component renders a Form composed by other Components inside a page.
+
+A common scenario for this component is to show a Form to create/edit records.
+
+- Properties:
+
+| Property         | Direction | Data Type | Description                                                                               |
+| ---------------- | --------- | --------- | ----------------------------------------------------------------------------------------- |
+| definition       | Inbound   | Reference | The modeled entity the Form is referencing                                                |
+| dataSource       | Inbound   | Text      | The dataSource instance where the entity is being created/edited                          |
+| code             | Inbound   | Text      | The instance of the definition being edited                                               |
+| data             | Outbound  | Object    | The instance data contained in the form component                                         |
+| hasChanges       | Outbound  | Boolean   | Flag to indicate if the data contained in the form has changed                            |
+| useTemporary     | Inbound   | Boolean   | Flag to indicate if the create/edit operation is using a Temporary                        |
+| hasErrors        | Outbound  | Boolean   | Flag to indicate if the form has errors                                                   |
+| state            | Outbound  | Object    | The state machine state of the entity                                                     |
```

Comment:
Type types of all known objects

Reply:
Describe known types on a separate chapter